addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1188,7 +1188,6 @@ var SnippetEditor = Widget.extend({
                     } else {
                         gridUtils._setElementToMaxZindex(self.$target[0], rowEl);
                     }
-                    gridUtils._setElementToMaxZindex(backgroundGridEl, rowEl);
```

Comment:
The fact that the grid was placed in front of the elements was useful for cases where the grid would be so full we would not see it anymore (e.g. the Masonry and Big Boxes snippets, or any grid where the grid items would have a background) so this behavior should be restored :slightly_smiling_face: Even if there is the blue drag helper to see where to drop, the grid should still be visible and since it is now fully transparent, it should not "obstruct the view" anymore :slightly_smiling_face: 

Note that it should also be restored in the `snippets.options.js` file :)

Reply:
I understand your point of view, but I'm not sure that the grid should always be in front of the elements 🤔 .

The grid is mainly an indicator to help you understand which mode you're in.
You don't need the grid to move your elements precisely, because your movements are already locked to the grid.
Also, if I'm not mistaken, "Big Boxes" is the only snippet where elements cover the entire snippet by default. You don't need the grid to understand that you can drag handles and edges to move and modify your elements.

> the grid should still be visible and since it is now fully transparent, it should not "obstruct the view" anymore

The grid is not fully transparent, it uses opacity but remains too visible when placed in front of elements. This was made on purpose to be sure that the grid remains visible in all cases, on the background image of a Banner for example 🙂 .

| Readable  | Less readable |
| ------------- | ------------- |
| ![Capture d’écran 2023-09-18 à 15 47 37 (2)](https://github.com/odoo/odoo/assets/80679690/d4ee4fe1-a267-4ea9-a6c6-489bd093eabe)  | ![Capture d’écran 2023-09-18 à 15 49 59 (2)](https://github.com/odoo/odoo/assets/80679690/b460a88c-2979-4484-b861-b737070e9f07)

By the way, Squarespace does exactly the same thing : the grid is behind the elements and is no longer visible when you cover it. This is probably done on purpose 🙂 

|  |  |
| ------------- | ------------- |
| ![Capture d’écran 2023-09-18 à 16 30 54](https://github.com/odoo/odoo/assets/80679690/2b632830-015f-45e4-ae27-269e7eef5c57)  | ![Capture d’écran 2023-09-18 à 15 46 25](https://github.com/odoo/odoo/assets/80679690/370e61b5-93ef-4ee8-8e17-d27db472a3a4)

But maybe I'm wrong 😅, what do you think @pierre-pde ? Do you prefer the grid in front of or behind elements ?

Reply:
Good point that it is the same behavior in squarespace but it still does not feel right to me either: if we don't need the grid (either functionally or aesthetically), why do we bother having it at all? In the Masonry snippet case, it feels not right to me either, there are blocks  covering the whole thing, you can move them and for some reason there is a piece of grid behind them... but same as in your squarespace screenshot I guess :grimacing: 

Decision for you @pierre-pde :)

Reply:
To be clear, I don't mind either way but: "show it everywhere" or "never show it" feels more consistent than "show it sometimes". But maybe this is not about consistency :smile:

Reply:
@Brieuc-brd :
We had a quick chat with Soukeina:
1. I agree with Quentin & Soukeina that, because we have building blocks that cover entirely the grid (Masonry & Big boxes), it's probably **not** a good idea to do like Squarespace: the grid behind all elements.
2. We thought of the solution "put the grid in the second position, behind selected element". The problem is that the order is then not respected anymore. The selected element appears always on top, and might go back behind other element(s) once dropped.
3. Soukeina proposed the following comprise; keeping the grid above all elements, but putting the grid opacity from 0.25 to 0.1, it looks great IMO :) 
![image](https://github.com/odoo/odoo/assets/75257428/f7c12051-adc7-45f4-b58a-8d1dc555faf2)
What do you think?
cc @qsm-odoo

Reply:
Indeed, it's easier to read than with an opacity of .25, and the grid still seems visible when there's a background image. Let's go with that, I'll update the branch 🙂